src/InputContainer.test.jsx

```diff
@@ -0,0 +1,64 @@
+/* 계획
+1. 아래 input 컴포넌트가 보인다
+2. 인풋을 change하고 click하는 기능이 바르게 작동한다
++3. 인풋의 value값을 바르게 받아온다
+*/
+import { fireEvent, render } from '@testing-library/react';
+import { useDispatch, useSelector } from 'react-redux';
+import InputContainer from './InputContainer';
+import { changeInput, registerInput } from './actions';
+import { initialState } from './reducer';
```

Comment:
```suggestion
import { fireEvent, render } from '@testing-library/react';

import { useDispatch, useSelector } from 'react-redux';

import InputContainer from './InputContainer';

import { changeInput, registerInput } from './actions';
import { initialState } from './reducer';
```

서로 관련이 있는 import끼리 묶어서 보여주면 좋을 것 같습니다.